src/views/MissionPlanningView.vue

```diff
@@ -534,6 +534,11 @@ const defaultNavCommands: MissionCommand[] = [
   },
 ]
 
+const makeDefaultNavCommands = (): MissionCommand[] => defaultNavCommandsTemplate.map((c) => ({ ...c }))
+
+const cloneCommands = (cmds?: MissionCommand[]): MissionCommand[] =>
+  cmds && cmds.length ? cmds.map((c) => ({ ...c })) : makeDefaultNavCommands()
```

Comment:
Can we make a "regular" if/else function out of this one? This kind-of-one-liner is unnecessarily hard to read.